src/components/Button/BaseButtonAddPost.js

```diff
@@ -101,7 +101,7 @@ const styles = StyleSheet.create({
   },
   text: {
     fontFamily: fonts.inter[500],
-    color: COLORS.white,
+    color: COLORS.almostBlack,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the text has been changed from `COLORS.white` to `COLORS.almostBlack`. Similar to the previous comment, ensure that this change doesn't affect the visibility and contrast of the text against its background, especially considering accessibility guidelines.
```diff
-     color: COLORS.white,
+     color: COLORS.almostBlack,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->